lib/td/client/model.rb

```diff
@@ -1,5 +1,14 @@
 require 'timeout'
 
+Process::CLOCK_MONOTONIC = Object.new unless Process.const_defined?(:CLOCK_MONOTONIC)
+def Process.clock_gettime(*arg)
```

Comment:
hmm. I think polyfill should be located in an independent file.

Currently `Process.clock_gettime` is used in this file only, but:

1. It would be used in another place in the future. That code requires `require "lib/td/model"` to use it on jruby 1.7. It will be invisible dependency to the developer.
2. When an another polyfill is needed, make a developer confusing where to put it